### ISSUE TITLE
Whisper stuck

### ISSUE THREAD

**Issue by eagerly-learn** [Const-me/Whisper#205]
Hi All... I'm a total newbie re: github and I;m not a coder or programmer.
I just teach basic 3D printing at a small semi-rural public library.
I used Dragon Naturally Speaking back in 2009 and really liked it but the disc won't load on my Windows 10 laptop.
MS wants $800 for DNS which I can't afford so I'm looking for a low cost to free voice to text software.
Whisper was recommended by a colleague but after downloading and clciking on the .exe file it keeps gettign stuck after press the Capture tab.
I'm unsure if i can post files or screenshots here as a newbie so I thought I'd ask respectfully.
Thank you.

**Comment by RickArcher108** [comment]
Hi "Eagerly". Your laptop may not have enough horsepower to run Whisper. How much material do you want to transcribe? If you like, I'll do it for you. No charge. No strings attached. Contact me at rick@batgap.com and we can discuss it.

**Comment by eagerly-learn** [comment]
Hi, I'm crazy busy at our small library handling a multitude of challenges so my focus has to change on the fly.

I wanted to use Whisper
[Whisper Stuck after Moel Request Run as Administrator.docx](https://github.com/Const-me/Whisper/files/14393649/Whisper.Stuck.after.Moel.Request.Run.as.Administrator.docx)
 as my voice to text so I could us a non-cloud based (downloadable) software.

DNS was great but now too pricey at $800.

I've attached what I see when I try to use Whisper on my Dell Latitude E6540 laptop running W10 with 8GB RAM.

Would you please share your suggestions of how I can correct this?

How can I ascertain the min specs for Whisper?

If my laptop lacks the specs is there another free program besides Whisper for V to T?

Thank you.

**Comment by tigros** [comment]
i looked up ur laptop so it's AMD FirePro W4170M which has only 2GB, almost for medium but not quite, try base or small models.

**Comment by eagerly-learn** [comment]
Hi tigros, thank you so much for checking on my laptop specs!  I'm not a programmer so please forgive my ignorance but I'm a little lost with your suggestion of ..."try base or small models."  I'm trying to figure out exactly what my next step(s) should be to make Whisper work on this laptop? Or are you intimating Whisper can't really work on this laptop and I need an updated laptop? Thank you again for your kind  help.

**Comment by RickArcher108** [comment]
I'm not a programmer either and Tigros has been very patient and helpful with me. You can download smaller models here: https://huggingface.co/getwavery/ggml-models/tree/main. The result will not be as accurate, but at least your laptop may be able to run the software.

**Comment by eagerly-learn** [comment]
Hi Rick.... I'm so grateful for everyone's kind help here on this wonderful forum. Thank you for sharing that link. I visited that site but I'm not  sure which of the ones on that list I should try installing and seeing it if works? Sorry for being such a Forest Gump. :)

**Comment by RickArcher108** [comment]
Try a few of the medium ones, such as [ggml-medium.bin](https://huggingface.co/getwavery/ggml-models/blob/main/ggml-medium.bin) and [ggml-small-distil-en.bin](https://huggingface.co/getwavery/ggml-models/blob/main/ggml-small-distil-en.bin). If neither of those work, try the small ones.

**Comment by eagerly-learn** [comment]
Hi Rick.  Thank you so much! I was reading here
https://www.wikihow.com/Open-BIN-Files you can't just open a .bin file in
Windows 10. Have you found one method of opening better than another? e.g.
creating a cue file -vs- burning an image to a DVD. So appreciate your help.

**Comment by tigros** [comment]
Hi,

you have 4 options on that page:

ggml-base-en.bin
ggml-base.bin
ggml-small-en.bin
ggml-small.bin

don't try to open them in windows, that's useless.

point the desktop GUI to use them.

you're welcome to try [Whisperer](https://github.com/tigros/Whisperer) if you have many files to process.

**Comment by eagerly-learn** [comment]
I really don't have a grasp on this all. I have no idea how you open these .en files once downloaded on to my laptop's hard drive except to open File Explore and find them on my C drive and click on it. That's how I've installed all my .exe files to run all sorts of open source software. Thank you.

**Comment by tigros** [comment]
that's not a .exe nor a .en, it's a .bin, as in binary, it's not executable, just data.

you have file extensions hidden, google how to unhide file extensions, im sure there's videos about it too.

when you run WhisperDesktop.exe it asks for model path, that's where you put the the file you just downloaded, a .bin

**Comment by RickArcher108** [comment]
Create a Whisper folder in the Program Files folder on your C drive and copy everything from the zip file into it. It should look something like this: 
![image](https://github.com/Const-me/Whisper/assets/131281569/a7110262-835d-4a1b-9a7e-5bc12d1e312f)
Then you should be able to click on WhisperDesktop.exe to open it.

**Comment by RickArcher108** [comment]
Tigros, today I tried to run Whisper instead of Whisperer, and got an error message, so I wanted to download a fresh version of it, and couldn't find where to do that. Please remind me.

**Comment by tigros** [comment]
WhisperDesktop? it's here https://github.com/Const-me/Whisper/releases

no new version since July 22.

**Comment by RickArcher108** [comment]
Eagerly, note that when you open Whisper Desktop you will see something like this:

![image](https://github.com/Const-me/Whisper/assets/131281569/51d1811c-aa78-4420-8891-670640867a52)

Click the three horizontal dots to the right of the box beneath "Model Path" and choose the ggml file you want to use, either medium or if that doesn't work, small. 

Then hit OK and that'll take you to the next dialog box where you can choose the mp3 file you want to transcribe, or you can use the Audio Capture feature:

![image](https://github.com/Const-me/Whisper/assets/131281569/e70873dc-01f2-467c-bcc5-539f49a41194)

**Comment by eagerly-learn** [comment]
I've never used github before so I have a learning curve.

I can see the .bin extension in File Explorer. I just don't know the
specific steps to "activate" (proper verb?) it.

Ex. I visit my Whisper folder I created to download Whisper.exe and I
clikon that .exe file and them Whisper loads but I get that error message
I've shared.

I really need the step by step of after I've boot up my W10 laptop on how
does one "implement" (correct verb?) to enable the .bin to work?

Thank you.

**Comment by eagerly-learn** [comment]
Yes, that's how I've had Whisper set up and  then I get that Whisper error
message.

Thank you.

**Comment by RickArcher108** [comment]
What error message are you getting? Keep in mind that you don't "open" the bin file. You link to it from within Whisper or Whisperer. You need to do that or they won't work.

**Comment by eagerly-learn** [comment]
Hi Rick,

Sorry for the late reply as I've had to resolve a plethora of tech issues.

II've attached the roor message I see when i tyr to run Whisper.

I'm just looking for a voice to text software I can talk into for
notes, emails as I i did easily with Dragon Naturally Speaking ver 9.

Thank you.

**Comment by RickArcher108** [comment]
You might want to try Livewriter:

![image](https://github.com/Const-me/Whisper/assets/131281569/dd44c7ae-6dd6-453c-96b4-029c3d05e168)

You can get it here: https://github.com/tigros/livewriter

**Comment by eagerly-learn** [comment]
I'm sorry as I'm still new with Github.

Would I download and install Whisper from here?

https://github.com/tigros/livewriter/blob/main/livewriter/Whisper.dll

Thank you kindly.

**Comment by RickArcher108** [comment]
You can get it from https://github.com/Const-me/Whisper/releases

I find GitHub confusing too. And Whisper can be tricky to install. If all you want to do is speech to text, I think you'll find Live Writer easier. You can get it here: https://github.com/tigros/livewriter

**Comment by eagerly-learn** [comment]
Hi Rick,

I'm so new to all this a sI used the DNS disc and would be up and running.

Shameful ow much M$$$$$ wants for it noe they they bought DNS.

Would you be so kind as to share with me which of these files in that
Whisper folder I would download and install?

[image: image.png]
Thank you ever so much.

Charles.

**Comment by RickArcher108** [comment]
To me, DNS means Domain Name Server. Other than that I don't know what you mean. You would download WhisperDesktop.zip. You'll also need to download one of the ggml files here: https://huggingface.co/getwavery/ggml-models/tree/main - base, small, medium, or large, depending on what your computer can handle. The larger the file, the slower it runs, but the more accurate the results. You'll also need to install Python: https://www.python.org/downloads/. 

There are people around here who are far more technically astute than I. I'll keep trying to help, but you may run into something that's beyond my minimal expertise.

**Comment by eagerly-learn** [comment]
Sorry. DNS is Dragon Naturally Speaking. It was developed by H/W team and
worked great. You installed from a disk and a Dilbert cartoon spent 45
minutes having you say words to train it. $100 and worth it but I can't get
Windows 10 to read the disk and now M$$$ wants $400-500 for it.

I don't have sophisticated needs - just transcribe my voice as I dictate
into written text.

I was hoping if the Windows works didn't have one that Linux might.  I
looked at Ubuntu Studio but that's not a feature.

I'll try my best with your kind help.

Honestly, did everyone suddenly become touch typists?

Thank you!

**Comment by RickArcher108** [comment]
As I said earlier, if all you want to do is speech to text, don't bother with Whisper. It's overkill. Install Live Writer. It's easier to install. You can get it here: https://github.com/tigros/livewriter. You'll still need to download one of the ggml files here: https://huggingface.co/getwavery/ggml-models/tree/main - base, small, medium, or large, depending on what your computer can handle.
